packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -14,30 +14,28 @@ import { TokenFromTokenlist } from "@/models/token";
 import { useAccount, useBalance, useNetwork } from "wagmi";
 import { formatUnits } from "viem";
 import { ModalId, useModalContext } from "@/contexts";
+import { FREQUENCY_OPTIONS } from "@/models/stack";
 
 interface SelectTokenButtonProps {
   label: string;
-  onClick: (isTokenFrom?: boolean) => void;
+  onClick: (isFromToken?: boolean) => void;
   token?: TokenFromTokenlist;
 }
 
-const HOUR_OPTION = "hour";
-const DAY_OPTION = "day";
-const WEEK_OPTION = "week";
-const MONTH_OPTION = "month";
-
 const frequencyOptions = [
-  { option: HOUR_OPTION, name: "Hour" },
-  { option: DAY_OPTION, name: "Day" },
-  { option: WEEK_OPTION, name: "Week" },
-  { option: MONTH_OPTION, name: "Month" },
+  { option: FREQUENCY_OPTIONS.hour, name: "Hour" },
+  { option: FREQUENCY_OPTIONS.day, name: "Day" },
+  { option: FREQUENCY_OPTIONS.week, name: "Week" },
+  { option: FREQUENCY_OPTIONS.month, name: "Month" },
 ];
```

Comment:
**Minor, non-blocking, suggestion that can be dismissed**
We may not need this one as we could simply use the `FREQUENCY_OPTIONS`

We could create a `titleCase` function to capitalize the first letter on a string into the utils folder (I'm assuming we would be using this one frequently) and just use the keys from the enum:
At line 220, where we use this variable:
```tsx
<div className="flex space-x-2">
  {Object.keys(FREQUENCY_OPTIONS).map((option) => {
    const isSelected = frequency === option;
    return (
      <RadioButton
        key={option}
        name={option}
        id={option}
        checked={isSelected}
        value={option}
        onChange={(event) =>
          setFrequency(event.target.value as FREQUENCY_OPTIONS)
        }
      >
        <BodyText
          size={2}
          className={cx({ "text-em-med": !isSelected})}
        >
          {titleCase(option)}
        </BodyText>
      </RadioButton>
    );
  })}
</div>
```

The `titleCase` util function could look something like this: 
```ts
export const titleCase = (word: string) =>
  word[0].toUpperCase() + word.slice(1).toLowerCase();
```

Reply:
Personally I think using hardcoded works better.

1. Easier to read and update
2. Less code = less bugs
3. Unnecessary JS running
4. We need to rewrite the logic if having translations or different names.

I think what you suggested  it's very handy, but I personally think it's better to keep it simple. What do you think?

Reply:
I don't have a strong opinion right now TBH. In my head, it made more sense to use the `enum` directly but the current app state feels the same. Right now, the util I suggested would be used in this file only.

Having i18n will imply fairly the same amount of work with either path we take IMHO.

So all in all I'd say if you're not quite convinced, dismiss these suggestions as doing so imply less work now :D

Marking this one as resolved 👌